tests/db/compact_db_migrations.py

```diff
@@ -0,0 +1,99 @@
+"""
+    Copyright 2023 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+
+
+    Tool to compact db migration scripts
+"""
+import asyncio
+import importlib
+import os
+import re
+
+import asyncpg
+import pytest
+
+UP_TO_VERSION = "202206290"  # Specify the version up to which migrations should be applied
```

Comment:
How did you get to this version? The latest ISO5 version is `5.4.2` and it has `v202209160.py` as it's latest database schema version for inmanta-core. Am I missing something?

Reply:
I went into the changelogs here: https://docs.inmanta.com/inmanta-service-orchestrator/latest/CHANGELOG.html
and I saw the latest release before ISO6 was 5.3.0. I think I went wrong there.